source/conf.py

```diff
@@ -60,3 +61,80 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# (Optional) Logo. Should be small enough to fit the navbar (ideally 24x24).
+# Path should be relative to the ``_static`` files directory.
+html_logo = "my_logo.png"
```

Comment:
Is this leftover? The latter may override this one.

Reply:
Great catch, fixed !